### DISCUSSION TITLE
What's the `subset` library?

### DISCUSSION THREAD

**Discussion by moegodot** [harfbuzz/harfbuzz#3738] [Q&A] [answered]
I noticed that if I install harfbuzz.I will get `libharfbuzz.a` and `libharfbuzz-subset.a`.

Should I link them at same time? What's the different from them?

**Comment by behdad** [comment, marked as answer]
The `harfbuzz-subset` library implements a font-subsetter. If you just want text-shaping, only link to `libharfbuzz`.
